i18n/en/docusaurus-theme-classic/footer.json

```
@@ -38,5 +38,9 @@
   "link.item.label.Qr Code e Push": {
     "message": "QR Code and Push",
     "description": "The label of footer link with label=Qr Code e Push linking to /"
+  },
+  "link.item.label.QR Code e Push": {
+    "message": "QR Code e Push",
+    "description": "The label of footer link with label=QR Code e Push linking to /"
```

Esse trecho me parece muito com o que está da linha 39 a 40. Se realmente for preciso ter, acredito que a precisa traduzir a preposição "e" para "and".

```suggestion
  "link.item.label.QR Code e Push": {
    "message": "QR Code and Push",
    "description": "The label of footer link with label=QR Code and Push linking to /"
```